conf/docker.config

```diff
@@ -16,6 +16,16 @@ docker {
 }
 
 process {
-  container = wf_container
+  container = 'nfcore/rnafusion:dev'
+
+  withName:star_fusion {
+    container = 'nfcore/rnafusion:star-fusion_v1.4.0'
```

Comment:
I can see that you tagged each tool with a different version.
I'm guessing it's the version of the tool.
Shouldn't it be easier to have a single tag for all tool, like the release from this pipeline for example?

Reply:
That would be a good idea, the problem is that some of the tools don't support `--version` parameter so you have no idea what version you are running. In this case, the `environment.yml` saves the day but for example, for `fusioncatcher` there is just commit message with a version. Based on this chose this approach instead. I think tagging it by pipeline release version can be confusing afterward.

Reply:
Alright, moving to proposed new semantic versioning. New convention is nfcore/rnafusion:<tool>_v<pipeline_version>